### PR TITLE
Add basic tests for defect_enhancement_task model

### PR DESCRIPTION
**Pull request by con-cat**
Addresses #438

### CODE REVIEW

File: tests/test_defect_enhancement_task.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+from bugbug.models.defect_enhancement_task import DefectEnhancementTaskModel
+
+
+def test_get_defect_enhancement_task_labels():
+    model = DefectEnhancementTaskModel()
+    classes, _ = model.get_labels()
+    assert classes[1042414] == "defect"
+    assert classes[1531080] == "task"
+    assert classes[1042734] != "enhancement"
```
Comment (marco-c) [inline tests/test_defect_enhancement_task.py]: Try to find a bug which is an enhancement instead

**Comment by con-cat** [comment]
I've added a bug that's an enhancement to bugs.json in the fixtures and updated the test to include that.

**Comment by marco-c** [comment]
> I've added a bug that's an enhancement to bugs.json in the fixtures and updated the test to include that.

There should already by some enhancement bug, can you remove your addition and just use one that is already there?

**Comment by con-cat** [comment]
It doesn't seem to be finding any enhancement bugs otherwise - here's the stdout call from the test with the existing data and a print statement added for classes:
```
---------------------------- Captured stdout call ----------------------------
14 defects
0 enhancements
2 tasks
{1042414: 'defect', 1042734: 'defect', 1046231: 'defect', 1047180: 'defect', 1053944: 'defect', 1087488: 'defect', 518272: 'defect', 528988: 'defect', 1037762: 'defect', 1042096: 'defect', 1488307: 'task', 1488310: 'defect', 1531080: 'task', 1388990: 'defect', 1325215: 'defect', 1364870: 'defect'}
```

**Comment by marco-c** [comment]
Oh, OK, that's unexpected :/
So can you add just the one you need, instead of two?

**Comment by con-cat** [comment]
I've now got just one new bug in the json file.

**Pull request merged**